Query Builder/moduleFunctions.php

```diff
@@ -18,33 +18,36 @@
 */
 
 //Returns are array with illeagel SQL keywords
-function getIllegals()
+function getIllegals($allowCommands = false)
```

Comment:
Is there merit in moving this into the gateway, along with the sole other function? Seeing as there is so much refactoring going on alreadu!

Reply:
Good call! I've made this change 👍